src/RpcClient/TransactionManager.cs

```diff
@@ -150,6 +148,21 @@ public TransactionManager AddMultiSig(KeyPair key, int m, params ECPoint[] publi
             return this;
         }
 
+        /// <summary>
+        /// Add Multi-Signature
+        /// </summary>
+        /// <param name="keys">The KeyPairs to sign transction</param>
+        /// <param name="m">The least count of signatures needed for multiple signature contract</param>
+        /// <param name="publicKeys">The Public Keys construct the multiple signature contract</param>
+        public TransactionManager AddMultiSig(KeyPair[] keys, int m, params ECPoint[] publicKeys)
+        {
+            foreach (var key in keys)
+            {
+                AddMultiSig(key, m, publicKeys);
```

Comment:
We should avoid the line 146 (https://github.com/neo-project/neo-modules/pull/213/files#diff-b0495f29c2f433fb99528ffde111175eR146), I think that it's better to duplicate the logic here.

Reply:
More keys can be added after this method, so I didn't check keys count in this method.